lib/matplotlib/axes/_base.py

```diff
@@ -4014,7 +4014,7 @@ def format_deltas(key, dx, dy):
         p = self._pan_start
         dx = x - p.x
         dy = y - p.y
-        if dx == 0 and dy == 0:
+        if dx == dy == 0:
```

Comment:
Oh, cool. I didn't realize you could do that with equals. I just always thought it was a special case in the interpreter for the `<` and `>` to do "between" tests.